packages/dicom-codec/src/codecs/codecFactory.js

```diff
@@ -0,0 +1,296 @@
+const logger = require("../utils/logger");
```

Comment:
Basically this file is where the magic goes.
Related Factory are used for some common methods for codecs js/wasm based.